### PR TITLE
Lesson 7 - Jupiter Class

### PR DESCRIPTION
**Pull request by Medinab27**
Boilerplate and content added that included name, about section, and experience and connect links section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```
Comment (lipecayres) [inline index.html:1]: It's smart to use the HTML template starter on IDE's. It gives us a template HTML to start with some metadata and helpful configs. If you are using VS Code, try typing the character ! in an empty HTML file and click the first option.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+<html>
+    <head>
+        <title>Barbara Medina's Portfolio</title>
+        <meta charset="utf-8">
+        <meta name= "author" content="Barbara">
+    </head>
+    <body>
```
Comment (lipecayres) [inline index.html:9]: Try to maintain HTML structure sintax using tags header/main/footer to build your section's

**Review by lipecayres — APPROVED** [review]
Great, you got all the requirements done! I added some tips in the code about how you can you code cleaner.

File: index.html (hunk 1 above)
Comment (Medinab27) [reply index.html:1]: I made a copy of my index.html file, erased the content in the copied version and then tried <! > but nothing came up. Perhaps I am misunderstanding the direction being given in the comment above. I can schedule a meeting for us to go over this and the other comments too.

File: index.html (hunk 2 above)
Comment (Medinab27) [reply index.html:9]: I am going to following something similar to this. https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Structuring_content/Structuring_documents 
I will add a footer at the bottom. 
And I will add a main after the body. We can also discuss if this is correct either at the next review or at a meeting I schedule with you.

**Pull request merged**